export const handleError = (e: Error) => {
  console.error(e);
  alert("🙀 오류가 발생하였습니다.")
}